prime-router/ktlint.asc

```diff
@@ -0,0 +1,17 @@
+-----BEGIN PGP SIGNATURE-----
```

Comment:
This doesn't look like it should be here.

Reply:
good catch. This was related to installing ktlint in my system. I just deleted it form the project.